h2o-py/h2o/model/binomial.py

```diff
@@ -852,6 +853,21 @@ def gains_lift(self, train=False, valid=False, xval=False):
         """
         return self._delegate_to_metrics('gains_lift', train, valid, xval)
 
+    @deprecated_params({'save_to_file': 'save_plot_path'})
+    def plot_gains_lift(self, type="both", xval=False, server=False, save_plot_path=None, plot=True, **kwargs):
```

Comment:
what are the `kwargs` for? they're not forwarded

Reply:
Thank you for noticing that. I must have copied the `kwargs` from some other method that used the `@deprecated_params` while assuming that the `kwargs` are used for the deprecated parameter which was a wrong assumption...